README.md

```diff
@@ -145,7 +153,9 @@ Chrome is pre-installed so that Django's selenium tests can be run in headless
 mode with a virtual display (id `:99`). For example, you may run a specific test
 like so:
 
-    (vm) $ runtests27-sqlite3 admin_widgets --selenium chrome
+    (vm) $ runtests36-sqlite3 admin_widgets --selenium chrome --parallel 1
+
+The test suite will sometimes hang when running selenium tests in parallel mode.
```

Comment:
I haven't seen this issue before but did reproduce it on the master branch just now. Did you see it on the 1.11 branch? I'm wondering if it's caused by a recent change in Django.